<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthToPCLClass</class>
 <widget class="QWidget" name="DepthToPCLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepthToPCL</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="ToolBarHorizontalLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,4,2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>高度系数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="HeightCoefficientSpinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ConfirmTransformationBtn">
             <property name="text">
              <string>确定变换</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="FramePickBtn">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>框选</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PointPickBtn">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>点选</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="ComfirmFramePickBtn">
             <property name="text">
              <string>确认框选区域</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ComfirmPointPickBtn">
             <property name="text">
              <string>确认点选区域</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="ViewYBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="shortcut">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ViewXBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ViewZBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="changeFormBtn">
         <property name="text">
          <string>转换3D/2D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawCounterBtn">
         <property name="text">
          <string>绘制轮廓</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="addDynamicLabel">
             <property name="text">
              <string>添加标签</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delDynamicLabel">
             <property name="text">
              <string>删除标签</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startTagBtn">
             <property name="text">
              <string>开始标记</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>242</width>
              <height>303</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="MarkCompleted">
             <property name="text">
              <string>完成标记</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearMarkersBtn">
             <property name="text">
              <string>清除标记内容</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drawMarkersBtn">
             <property name="text">
              <string>绘制标记</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VTKWidgetLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DepthToPCL.qrc"/>
 </resources>
 <connections/>
</ui>
